lib/Connection.js

```diff
@@ -20,14 +20,14 @@ const {
   iXmlNodeHead, iXmlNodeScriptOpen, iXmlNodeScriptClose,
 } = require('./ixml');
 
-const { iRestHttp } = require('./transports/irest');
-const { db2Call } = require('./transports/istoredp');
+const { httpCall } = require('./transports/http');
+const { idbCall } = require('./transports/idb');
```

Comment:
These should be `httpTransport` and `idbTransport` to match the others.

Reply:
Should be fixed now.